lib/application/pages/home/collections/update/update_collection_page.dart

```diff
@@ -173,3 +232,115 @@ class _MemosActionButton extends ConsumerWidget {
     );
   }
 }
+
+class _MemosReorderableList extends ConsumerWidget {
+  const _MemosReorderableList({
+    required this.memos,
+    required this.currentMemoIndex,
+    required this.onReorder,
+    this.onTap,
+  });
+
+  final List<MemoMetadata> memos;
+
+  /// Currently Memo being visualized when the list is opened.
+  ///
+  /// This will be highlighted in the list to indicate which Memo the user is seeing.
+  final int currentMemoIndex;
+
+  /// Triggers when a Memo is moved from `oldIndex` to `newIndex`.
```

Comment:
```suggestion
  /// Called when a memo is moved from [oldIndex] to [newIndex].
```
(not sure, but I think it can highlight the arg in function)

Reply:
It can't unfortunately